examples/api/lib/material/refresh_indicator/sliver_refresh_indicator.0.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for SliverRefreshIndicator
+
+import 'package:flutter/material.dart';
+
```

Comment:
The failing framework test shard is due to this sample code see logs below:

```
The following TestFailure was thrown running a test:
Expected: exactly one matching node in the widget tree
  Actual: _WidgetTypeFinder:<zero widgets with type "WidgetsApp" (ignoring offstage widgets)>
   Which: means none were found but one was expected
When the exception was thrown, this was the stack:
#4      main.<anonymous closure> (file:///tmp/flutter%20sdk/examples/api/integration_test/smoke_integration_test.dart:2457:9)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)
This was caught by the test expectation on the following line:
  file:///tmp/flutter%20sdk/examples/api/integration_test/smoke_integration_test.dart line 2457
The test description was:
  Smoke test material/refresh_indicator/sliver_refresh_indicator.0.dart
```

Reply:
I would refer to other sample code to see if there is something missing from your case here.